sbt-metals/src/sbt-test/sbt-metals/semanticdb/build.sbt

```diff
@@ -26,11 +26,11 @@ lazy val b = project
     inConfig(Test) { checkSemanticdb := assertSemanticdbForScala3.value },
   )
 
-// old scala version
+// scala 2.12
 lazy val c = project
   .in(file("c"))
   .settings(
-    scalaVersion := "2.12.8", // latest scalameta doesn't publish for ~2.12.7
+    scalaVersion := "2.12.17",
```

Comment:
Ach ok, I thought that was on purpose to use the last supported version, but I don't see are reason to :thinking: